VVResonances/interactive/makeInputs.py

```diff
@@ -150,11 +143,19 @@ def makeSignalYields(filename,template,branchingFraction,sfP = {'HPHP':1.0,'HPLP
   cmd='vvMakeSignalYields.py -s {template} -c "{cut}" -o {output} -V "jj_LV_mass" -m {minMVV} -M {maxMVV} -f {fnc} -b {BR} --minMX {minMX} --maxMX {maxMX} samples'.format(template=template, cut=cut, output=yieldFile,minMVV=minMVV,maxMVV=maxMVV,fnc=fnc,BR=branchingFraction,minMX=minMX,maxMX=maxMX)
   os.system(cmd)
 
+def fitVJets(filename,template):
+  for p in purities:
+    cut='*'.join([cuts['common'],cuts[p],cuts['acceptance']])
+    rootFile=filename+"_"+p+".root"
+    cmd='vvMakeVjetsShapes.py -s "{template}" -c "{cut}"  -o "{rootFile}" -m {minMJ} -M {maxMJ} --store "{filename}_{purity}.py" --minMVV {minMVV} --maxMVV {maxMVV} samples'.format(template=template,cut=cut,rootFile=rootFile,minMJ=minMJ,maxMJ=maxMJ,fixPars=fixPars,filename=filename,purity=p,minMVV=minMVV,maxMVV=maxMVV)
```

Comment:
I removed it already from the cmd (it's not in the line above). I now set it = "1" by default here: https://github.com/jngadiub/cmgtools-lite/blob/3DVVFit/VVResonances/scripts/vvMakeVjetsShapes.py#L24
because with default value "" as you were doing it does not work as the parser interprets the next string as its value. Or it might still work after I added this check here: https://github.com/jngadiub/cmgtools-lite/blob/3DVVFit/VVResonances/scripts/vvMakeVjetsShapes.py#L99

Reply:
no i mean in this line there is a ,fixPars=fixPars, so we either have to remove that or add it back so that it works :-)

Reply:
ah now I see what you mean :) let me change it.